testapps/test_elasticsearch/tests/test_prod_es_indices.py

```diff
@@ -35,7 +35,7 @@ def test_prod_config(self):
     },
     {
         "alias": "hqapps",
-        "index": "test_hqapps_2016-07-11_2128",
+        "index": "hqapps_2016-10-20_1835",
```

Comment:
Builds finish quicker when boston devs are asleep :). So making this quick change since this seems to be causing the build failure and blocking this.